diff --git a/dist/node/operations/createDirectory.js b/dist/node/operations/createDirectory.js
index 27c7cf8520f550c7608ce2c6f1add6c3d1128960..8f707536a4e03315e7d0ba4aad46e1ad3bbb3c3b 100644
--- a/dist/node/operations/createDirectory.js
+++ b/dist/node/operations/createDirectory.js
@@ -54,7 +54,7 @@ async function createDirectoryRecursively(context, dirPath, options = {}) {
         }
         catch (err) {
             const error = err;
-            if (error.status === 404) {
+            if (error.status === 404 || error.status === 409) {
                 creating = true;
                 await createDirectory(context, testPath, {
                     ...options,
diff --git a/dist/node/operations/exists.js b/dist/node/operations/exists.js
index 12095a17b664e508ad63c45d0820207c476f2e0f..865a31b0916e0bce137ffa461600d53c48638558 100644
--- a/dist/node/operations/exists.js
+++ b/dist/node/operations/exists.js
@@ -5,7 +5,7 @@ export async function exists(context, remotePath, options = {}) {
         return true;
     }
     catch (err) {
-        if (err.status === 404) {
+        if (err.status === 404 || err.status === 409) {
             return false;
         }
         throw err;
